README.rst

```diff
@@ -38,6 +38,11 @@ Use ``make`` to check that you are using correct
 `Markdown <https://pandoc.org/MANUAL.html#pandocs-markdown>`__ syntax,
 and double-check the generated ``draft-*.html`` file before filing a Pull Request.
 
+This repo uses devcontainer.json to ease configuring the tools necessary to run ``make``.
+You can use [Codespaces](https://docs.github.com/en/codespaces/developing-in-codespaces/developing-in-a-codespace)
+to make your contributions and validate your work.
+Or when using VS Code, use the "Dev Containers: Reopen in Container" command to create
+the docker container with the tools.
```

Comment:
`README.rst` is a generated file; please edit `README.template` instead and run `make`. As suggested above, this should preferably be a link to another documentation file (say `devcontainer.rst`) to keep the README uncluttered. Please include info for people to be able to tell whether they want to use this directly in the README, though.

Reply:
Is there any particular reason to use `devcontainer.rst` as opposed to the more common `.md` format for this new file?